crates/starknet-server/src/api/http/endpoints/blocks.rs

```diff
@@ -1,11 +1,39 @@
-use axum::Json;
+use axum::{Extension, Json};
 
 use crate::api::http::error::HttpApiError;
-use crate::api::http::models::{AbortedBlocks, AbortingBlocks, CreatedBlock};
-use crate::api::http::HttpApiResult;
+use crate::api::http::models::{AbortedBlocks, AbortingBlocks, BlockInfo};
+use crate::api::http::{HttpApiHandler, HttpApiResult};
 
-pub(crate) async fn create_block() -> HttpApiResult<Json<CreatedBlock>> {
-    Err(HttpApiError::GeneralError)
+pub(crate) async fn create_block(
+    Extension(state): Extension<HttpApiHandler>,
+) -> HttpApiResult<Json<BlockInfo>> {
+    let mut starknet = state.api.starknet.write().await;
+    starknet.update_pending_block_timestamp();
+    let _ = starknet.create_block().map_err(|_| HttpApiError::CreateEmptyBlockError);
+    let last_block = starknet.get_latest_block();
```

Comment:
there is already a method get_block, which accepts BlockId as parameter

Reply:
yes it was fixed in 48015d28db3fce9bfd6924c695e918ad7ae342a7